content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -211,13 +211,15 @@ Once Pod is assigned to a node by scheduler, kubelet starts creating containers
 
 {{< feature-state for_k8s_version="v1.14" state="stable" >}}
 
-In order to add extensibility to Pod readiness by enabling the injection of
-extra feedback or signals into `PodStatus`, Kubernetes 1.11 introduced a
-feature named [Pod ready++](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0007-pod-ready%2B%2B.md).
-You can use the new field `ReadinessGate` in the `PodSpec` to specify additional
-conditions to be evaluated for Pod readiness. If Kubernetes cannot find such a
-condition in the `status.conditions` field of a Pod, the status of the condition 
-is default to "`False`". Below is an example:
+Your application can inject extra feedback or signals into `PodStatus`, using a
+feature named feature named _Pod ready++_. To use this, set `ReadinessGate` in the
+`PodSpec` to specify additional conditions to be evaluated for Pod readiness.
```

Comment:
@sftim , line 215, clean up double `feature named`. Is there other information that you could link to (is this feature listed in the feature gates table)?
nit: line 215, Suggested edit: 
To use this feature [or To use the Pod ready++ feature], you can set `ReadinessGate` in the `PodSpec`.
The `ReadinessGate` field lists additional Pod readiness conditions.